# This patch fixes a problem with sdl_ttf which causes pydance to crash on startup
# Priyank <priyankmg@gmail.com>

--- fontfx.py	 2006-06-04 17:51:30.000000000 +0530
+++ fontfx.py	 2004-05-23 21:30:42.000000000 +0530
@@ -144,7 +144,7 @@
     self.bgcolor = bgcolor
     self.offcolor = [c^40 for c in color]
     self.notcolor = [c^0xFF for c in color]
-    self.text = font.render(message, False, [255, 0, 0], self.notcolor)
+    self.text = font.render(message, True, [255, 0, 0], self.notcolor)
     self.text.set_colorkey(1, RLEACCEL)
     self.outline = self.textHollow(font, message, color)
     self.bar = pygame.Surface(self.text.get_size())
@@ -155,7 +155,7 @@
     self.ratio = width / 100.0
 
   def textHollow(self, font, message, fontcolor):
-    base = font.render(message, 0, fontcolor, self.notcolor)
+    base = font.render(message, 1, fontcolor, self.notcolor)
     size = base.get_width() + 2, base.get_height() + 2
     img = pygame.Surface(size, 16)
     img.fill(self.notcolor)
